nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/test/java/org/apache/nifi/controller/repository/TestWriteAheadFlowFileRepository.java

```diff
@@ -74,6 +86,295 @@ public void clearRepo() throws IOException {
         }
     }
 
+
+    @Test
+    public void testUpdatePerformance() throws IOException, InterruptedException {
```

Comment:
I don't think you meant to leave this test as is without @Ignore. It certainly takes time to complete.